src/sc_crawler/alembic/versions/a9b0c1d2e3f4_v0_9_0_merge_benchmark_score_tables.py

```
@@ -0,0 +1,734 @@
+            table_name,
+            list(primary_keys),
+        )
+
+    if is_postgresql:
+        op.execute(
+            sa.text(
+                f"""
+                UPDATE {table_name}
+                SET environment = jsonb_build_object('kernel_version', kernel_version)
+                WHERE kernel_version IS NOT NULL
+                """
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)